tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_acme.yml

```diff
@@ -0,0 +1,36 @@
+- name: Installing ACME Responder
+  shell: pki-server acme-create --instance {{topology}}-CA
+  notify:
+    - INC_CONSTANTS
+
+- meta: flush_handlers
+
+- name: Configuring ACME Responder Database
+  copy:
+    src: /usr/share/pki/acme/conf/database/in-memory/database.conf
```

Comment:
The file was moved to `/usr/share/pki/acme/database/in-memory/database.conf`, but shouldn't we use a persistent database instead to mimic a real deployment, e.g. `/usr/share/pki/acme/database/ldap/database.conf`? If you use LDAP, there's some additional steps required:
https://github.com/dogtagpki/pki/blob/master/docs/installation/acme/Configuring_ACME_Database.md

Reply:
Fixed